components/site/src/lib.rs

```diff
@@ -1114,7 +1115,11 @@ impl Site {
         }
 
         if let Some(ref redirect_to) = section.meta.redirect_to {
-            let permalink = self.config.make_permalink(redirect_to);
+            let permalink: Cow<String> = if is_external_link(redirect_to) {
+                Cow::Borrowed(redirect_to)
```

Comment:
This should be added to the list of external links to check for

Reply:
I'm sorry, I don't understand what you're saying. By this, you mean `permalink`? Is there a list of external links somewhere? What would checking against it mean? I may be missing something critical here, since I'm not very familiar with Zola's codebase at the moment.

Reply:
So Zola checks external links in your markdown automatically to see if they are still valid. Since we will essentially redirect to this page, we should that URL to the list of external URLs to check so if you make a typo or the site is not up anymore you can correct/remove.

Have a look at `components/link_checker` to see where the checks happen